### ISSUE TITLE
Test failures (because of number accuracy?)

### ISSUE THREAD

**Issue by eserte** [gryphonshafer/Geo-GoogleEarth-AutoTour#1]
The CPAN Testers overview at http://matrix.cpantesters.org/?dist=Geo-GoogleEarth-AutoTour%201.03 shows only failures for freebsd and windows, and a mix of pass and fail for linux. Looking at the linux results one can see that all uselongdouble-compiled perls (these have "-ld" in the archname) are affected.

The diff in the test failures is difficult to read, but one possible reason is that numbers are slightly differing here due to floating point inaccuracies.

**Comment by gryphonshafer** [comment]
Sorry I've sat on this for such a long time. I appreciate your feedback. Any ideas on how to resolve this issue without too much pain?

**Comment by gryphonshafer** [comment]
Nevermind. Resolved (probably) with v1.04

**Comment by eserte** [comment]
It definitely looks better now: http://matrix.cpantesters.org/?dist=Geo-GoogleEarth-AutoTour%201.04
There are some failures with older perls, but this seems to be something different...

**Comment by eserte** [comment]
With 1.05 the test suite again fails on freebsd systems. A diff between "got" and "expected" shows the following:
```
@@ -1785,7 +1785,7 @@
 #           </Camera>
 #         </gx:FlyTo>
 #         <gx:FlyTo>
-#           <gx:duration>1.192231</gx:duration>
+#           <gx:duration>1.192230</gx:duration>
 #           <gx:flyToMode>smooth</gx:flyToMode>
 #           <Camera>
 #             <latitude>48.609944</latitude>
@@ -2241,7 +2241,7 @@
 #           </Camera>
 #         </gx:FlyTo>
 #         <gx:FlyTo>
-#           <gx:duration>1.353609</gx:duration>
+#           <gx:duration>1.353610</gx:duration>
 #           <gx:flyToMode>smooth</gx:flyToMode>
 #           <Camera>
 #             <latitude>48.521729</latitude>
```

**Comment by gryphonshafer** [comment]
Resolved in v1.06
